runtime-modules/bounty/src/benchmarking.rs

```diff
@@ -172,66 +190,181 @@ benchmarks! {
             judging_period: One::one(),
             cherry,
             creator: creator.clone(),
+            max_amount,
             ..Default::default()
         };
 
-        Module::<T>::create_bounty(
+        Bounty::<T>::create_bounty(
             RawOrigin::Signed(account_id.clone()).into(),
             params,
             Vec::new()
         ).unwrap();
 
-        let bounty_id: T::BountyId = Module::<T>::bounty_count().into();
+        let bounty_id: T::BountyId = Bounty::<T>::bounty_count().into();
 
         assert!(Bounties::<T>::contains_key(bounty_id));
     }: cancel_bounty(RawOrigin::Signed(account_id), creator.clone(), bounty_id)
     verify {
-        let bounty = <crate::Bounties<T>>::get(bounty_id);
+        let bounty = Bounty::<T>::bounties(bounty_id);
 
         assert!(matches!(bounty.milestone, BountyMilestone::Canceled));
         assert_last_event::<T>(Event::<T>::BountyCanceled(bounty_id, creator).into());
     }
 
     veto_bounty {
+        let max_amount: BalanceOf<T> = 1000.into();
+
         let params = BountyCreationParameters::<T>{
             work_period: One::one(),
             judging_period: One::one(),
+            max_amount,
             ..Default::default()
         };
 
-        Module::<T>::create_bounty(RawOrigin::Root.into(), params, Vec::new()).unwrap();
+        Bounty::<T>::create_bounty(RawOrigin::Root.into(), params, Vec::new()).unwrap();
 
-        let bounty_id: T::BountyId = Module::<T>::bounty_count().into();
+        let bounty_id: T::BountyId = Bounty::<T>::bounty_count().into();
 
         assert!(Bounties::<T>::contains_key(bounty_id));
     }: _ (RawOrigin::Root, bounty_id)
     verify {
-        let bounty = <crate::Bounties<T>>::get(bounty_id);
+        let bounty = Bounty::<T>::bounties(bounty_id);
 
         assert!(matches!(bounty.milestone, BountyMilestone::Canceled));
         assert_last_event::<T>(Event::<T>::BountyVetoed(bounty_id).into());
     }
 
     fund_bounty {
+        let max_amount: BalanceOf<T> = 100.into();
```

Comment:
Shouldn't we have a benchmark for each branch of `get_bounty_milestone_on_funding`?

As I understand currently we are only benchmarking the case `maximum_funding_reached` is true. Am I right?

Reply:
This is the worst-case scenario because it fires an additional event. Other branches are similar.